elasticdl/python/tests/worker_trainloop_pytorch_test.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2020 The ElasticDL Authors. All rights reserved.
```

Comment:
The filename "worker_pytorch_training_loop_test" may be better.

Reply:
good suggestion!